test-prow-e2e.sh

```diff
@@ -32,6 +32,8 @@ export DBUS_SESSION_BUS_ADDRESS
 
 SCENARIO="${1:-e2e}"
 
+node ./test-count.js
+
```

Comment:
Doesn't this count cypress test results? But the cypress are executed below?! :man_shrugging: